README.md

```diff
@@ -31,16 +31,50 @@ npm install --save-dev spectron
 Spectron works with any testing framework but the following example uses
 [mocha](https://mochajs.org):
 
+
+## To use
+To get up and running from your command line:
+```sh
+# Install mocha globally. 
+npm i mocha -g 
```

Comment:
I have this thing against global module installations.  Especially when it comes to commonly used tools like mocha where projects could depend on different versions